Revision to neonatal intervention docs (#1598)
ihmeuw/vivarium_research: docs/source/models/intervention_models/neonatal/cpap_intervention.rst

    - Adjust multiplicatively using RR
    - Yes
    - For convenience, we will model this like a dichotomous risk factor; more details below

Baseline Coverage and RR Data
+++++++++++++++++++++++++++++

39.3% of CEmONC facilities and 7.5% of BEmONC facilities have CPAP, according to the 2016 Ethiopia EmONC Final Report.  Please use these as a placeholder for now while we try to find reliable values for Nigeria and Pakistan. 

We might be able to borrow strength from other locations and times by predicting coverage for more country-years simultaneously, perhaps even in combination with other key intervention technologies, based on sources such as existing `Service Provision Assessment (SPA) <https://www.dhsprogram.com/methodology/Survey-Types/SPA.cfm>`_ and `Service Availability and Readiness Assessment (SARA) <https://www.who.int/data/data-collection-tools/service-availability-and-readiness-assessment-(sara)>`_ data.

.. list-table:: Baseline Coverage of CPAP (placeholder values)
  :widths: 15 15 15 15
  :header-rows: 1

  * - Birth Facility
    - Coverage Mean (%)
    - Coverage Distribution (%)
    - Notes
  * - Home Birth
    - 0
    - Deterministic value (no uncertainty)

>     - Assumtion; need to double check if this is reasonable

Reviewer: ```suggestion
    - Assumption; need to double check if this is reasonable
```

where :math:`p(\text{RDS})` is the probability of dying from Preterm with RDS in the general population, and :math:`p(\text{RDS}|\text{CPAP})` and :math:`p(\text{RDS}|\text{no CPAP})` are the probability of dying from Preterm with RDS in setting with and without access to CPAP.  For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has CPAP available (7.5% in Ethiopia in 2016)

> When we fill in the location-specific values for delivery facility rates, CPAP coverage by facility type, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (p_RDS_w_CPAP and p_RDS_wo_CPAP), which we can solve analytically.

Reviewer: ```suggestion
When we fill in the location-specific values for delivery facility rates, CPAP coverage by facility type, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{RDS}|\text{CPAP})` and :math:`p(\text{RDS}|\text{no CPAP})`), which we can solve analytically.
```

  p_CPAP_CEmONC = 0.393
  RR_CPAP = 0.53

  p_RDS_w_CPAP = ... # solve system of equations from previous section
  p_RDS_wo_CPAP = # TODO: fill this in explicitly

  RR_no_CPAP = 1 / RR_CPAP
  # p_RDS_w_CPAP = (1 - PAF_no_CPAP) * p_RDS
  PAF_no_CPAP = 1 - p_RDS_w / p_RDS # solved equation in previous line for PAF

>   PAF_no_CPAP = 1 - 1 / (RR_no_CPAP * p_RDS_wo_CPAP + p_RDS_w_CPAP)

Reviewer: This isn't right, is it? The coverage of CPAP doesn't seem to be entering in anywhere here...

Author: It is part of the calculation of p_RDS_wo_CPAP, e.g. if coverage of CPAP is zero, then p_RDS_wo_CPAP = p_RDS.

Reviewer: But this equation for the PAF of no-CPAP does not appear to have any term in the right hand side that includes the coverage. It seems like it should be 
```suggestion
  PAF_no_CPAP = 1 - 1 / (RR_no_CPAP * p_no_CPAP + p_CPAP)
```

Reviewer: And I don't understand how it relates to the previous two lines -- is it an alternative?

Author: I think this is an attempt at the alternative equation that Nathaniel proposed, and that I made the centerpiece of the exposition for antibiotics.

  p_CPAP_CEmONC = 0.393
  RR_CPAP = 0.53

  p_RDS_w_CPAP = ... # solve system of equations from previous section
  p_RDS_wo_CPAP = # TODO: fill this in explicitly

  RR_no_CPAP = 1 / RR_CPAP
  # p_RDS_w_CPAP = (1 - PAF_no_CPAP) * p_RDS
  PAF_no_CPAP = 1 - p_RDS_w / p_RDS # solved equation in previous line for PAF

**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that :math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.
The math for this can be found :ref:`on the antibiotics page <intervention_neonatal_antibiotics>`

> and the pseudocode would look as follows:
> 
> ::

Reviewer: ```suggestion
and the pseudocode would look as follows::
```
